src/style.css

```diff
@@ -1,3 +1,112 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-bold {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.logo_link {
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  margin-right: 50px;
+  padding: 0;
+  height: 60px;
+}
+
+.u-list {
+  margin: 0;
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+}
+
+.is-active_image {
+  position: relative;
+  bottom: 4;
+}
+
+.nav_list_item {
+  list-style-type: none;
+  padding: 0;
+}
+
+.nav_link {
+  margin: 0 10px;
+  line-height: 60px;
+
+  /* margin-right: 10px; */
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  font-size: 12px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.nav_link:hover {
+  color: var(--blue);
+}
+
+.is-active {
+  display: flex;
+  flex-direction: column;
+  color: var(--blue);
+  margin-left: 0;
+  text-align: left;
+}
+
+.is-active::after {
+  position: relative;
+  background-image: url(./images/Rectangle.png);
+  bottom: 4;
+}
+
+.video {
```

Comment:
Don't hardcode the name of the class according to the link name. What if it changes?

Reply:
Can inspect my new code?